VideoLocker/res/values/strings.xml

```diff
@@ -1,7 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="app_name">edX</string>
```

Comment:
(Hmm, looks like that's because you readded it though, so I guess I was right after all)